operator/cmd/mesh/testdata/manifest-generate/data-snapshot/charts/istiod-remote/templates/mutatingwebhook.yaml

```diff
@@ -16,7 +16,7 @@ webhooks:
   - name: sidecar-injector.istio.io
     clientConfig:
       {{- if .Values.istiodRemote.injectionURL }}
-      url: {{ .Values.istiodRemote.injectionURL }}
+      url: {{ .Values.istiodRemote.injectionURL }}/{{ .Values.global.multiCluster.clusterName }}/{{ .Values.global.network }}
```

Comment:
Same comment as in all other PRs: please use mesh config / proxy config, or if it doesn't make sense add a comment in values explaining why it's not possible. 

What is the meaning if 'global.multiCluster.clusterName' ? It can't be global since each cluster has a different name. Neither network - it doesn't make sense to call it global. 

Can we have the clusterName and network as params in istiod-remote only, without mixing global ? 

In general - global is the worse in terms of pollution and missuse.

Reply:
@costinm I'm not introducing any new ways to configure multicluster cluster name or network.  This is just to leverage the configurations provided by our existing docs on shared control plane - https://istio.io/docs/setup/install/multicluster/shared/#remote-cluster

I agree these needs optimization as it doesn't make sense to be global when the content is cluster specific.  The optimization probably deserves a doc and separate PRs to clean up.